src/index.ts

```diff
@@ -84,7 +84,7 @@ export default function traverse(
   // thus adding it to the recursive stack will prevent it from being explored if the
   // boolean is seen in a further nested schema.
   if(depth === 0) {
```

Comment:
```suggestion
  if (depth === 0) {
```

hmm linting should have caught this...